ui/lib/sync/addon/components/secrets/page/overview.hbs

```diff
@@ -20,7 +20,119 @@
       </ul>
     </nav>
   </div>
-  {{! overview cards here }}
+  <Toolbar>
+    <ToolbarActions>
+      <ToolbarLink @route="secrets.destinations.create" @type="add" data-test-create-destination>
+        Create new destination
+      </ToolbarLink>
+    </ToolbarActions>
+  </Toolbar>
+
+  <OverviewCard @cardTitle="Secrets by destination" class="has-top-margin-l">
+    {{#if this.fetchAssociationsForDestinations.isRunning}}
+      <div data-test-sync-overview-loading>
+        <Icon @name="loading" @size="24" />
+        Loading destinations...
+      </div>
+    {{else if (not this.destinationMetrics)}}
+      <EmptyState
+        @title="Error fetching information"
+        @message="Ensure that the policy has access to read sync associations."
+      >
+        <DocLink @path="/vault/api-docs/system/secrets-sync#read-associations">
+          API reference
+        </DocLink>
+      </EmptyState>
+    {{else}}
+      <Hds::Table>
+        <:head as |H|>
+          <H.Tr>
+            <H.Th>Sync destination</H.Th>
+            <H.Th @align="right"># of secrets</H.Th>
+            <H.Th>Last updated</H.Th>
+            <H.Th @align="right">Actions</H.Th>
+          </H.Tr>
+        </:head>
+        <:body as |B|>
+          {{#each this.destinationMetrics as |data index|}}
+            <B.Tr data-test-overview-table-row>
+              <B.Td>
+                <Icon @name={{data.icon}} data-test-overview-table-icon={{index}} />
+                <span data-test-overview-table-name={{index}}>{{data.name}}</span>
+                <Hds::Badge
+                  @text={{data.status}}
+                  @color={{if (eq data.status "All synced") "success"}}
+                  data-test-overview-table-badge={{index}}
+                />
+              </B.Td>
+              <B.Td @align="right" data-test-overview-table-total={{index}}>{{data.associationCount}}</B.Td>
+              <B.Td data-test-overview-table-updated={{index}}>{{date-format data.lastSync "MMMM do yyyy, h:mm:ss a"}}</B.Td>
```

Comment:
Do we want a `-` or something if there is no `lastSync` value?

Reply:
<img width="1130" alt="Screenshot 2023-12-04 at 11 50 06 AM" src="https://github.com/hashicorp/vault/assets/68122737/fbeb6a4b-25df-4062-b052-3cb62613c09f">

Reply:
Good catch! Let me check what the HDS guidance is on placeholders (-, n/a etc.) and get that updated.

Reply:
Updated to display an emdash.